src/bin/bench.rs

```diff
@@ -26,7 +26,7 @@ pub const USAGE: &'static str = "
 Execute all benchmarks of a local package
 
 Usage:
-    cargo bench [options] [--] [<args>...]
+    cargo bench [options] [-p SPEC --package SPEC]... [--] [<args>...]
```

Comment:
I think that new syntax in docopt may actually allow for nicer definitions of this, something like:

```
Options:
    -p, --package=SPEC ...         Description ...
```

I believe it's still somewhat experimental, but this may be a good place to get a good start on it?